### PR TITLE
add eido

### PR DESCRIPTION
**Pull request by stolarczyk**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by stolarczyk** [comment]
wait for #12285

**Comment by stolarczyk** [comment]
@conda-forge/staged-recipes ready for review

**Comment by chrisburr** [comment]
As it failed due to `peppy` not being uploaded yet:

@conda-forge-admin, please restart ci

File: recipes/eido/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  run:
+    - python
+    - ubiquerg >=0.5.2
+    - logmuse >=0.2.5
+    - jsonschema >=3.0.1
+    - peppy >=0.30.1
+    - yacman >=0.6.7
+
+test:
+  imports:
```
Comment (chrisburr) [inline recipes/eido/meta.yaml:32]:
```suggestion
  requires:
    - pip
  imports:
```

File: recipes/eido/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    - eido
+  commands:
+    - eido --help
```
Comment (chrisburr) [inline recipes/eido/meta.yaml:35]:
```suggestion
    - eido --help
    - pip check
```

**Pull request merged**